### ISSUE TITLE
ESLint Error: Cannot read property 'kind' of null

### ISSUE THREAD

**Issue by kevanstannard** [benmosher/eslint-plugin-import#841]
I'm currently using airbnb lint rules plus flow typing with the following configuration:

**package.json**

~~~
"babel-eslint": "^7.2.3",
"eslint": "^3.19.0",
"eslint-config-airbnb": "^15.0.1",
"eslint-plugin-flowtype": "^2.33.0",
"eslint-plugin-import": "^2.2.0",
"eslint-plugin-jsx-a11y": "^5.0.3",
"eslint-plugin-react": "^7.0.1"
~~~

**.eslintrc**

~~~yaml
---
  extends:
    - airbnb
    - plugin:flowtype/recommended
  plugins:
    - flowtype
  parser: babel-eslint
~~~

And just one file in the project:

**types.js**

~~~js
type Person = {
  name: string
}

export type {
  Person
};
~~~

When running eslint on this file it generates the following error:

~~~
Cannot read property 'kind' of null
TypeError: Cannot read property 'kind' of null
    at checkDeclaration (/Users/Kevan/dev/lint-bug/node_modules/eslint-plugin-import/lib/rules/no-mutable-exports.js:10:24)
~~~

If I change the `types.js` file as follows, then the error goes away:

~~~js
export type Person = {
  name: string
}
~~~

I'm not sure if this is best solved within this project or within airbnb but just raising it here first as the error is occurring within the import plugin code.

Thanks

**Comment by jseminck** [comment]
Hi. It looks like this issue is already resolved in the master branch, but not yet released.

Here is the commit that fixed the issue:

https://github.com/benmosher/eslint-plugin-import/commit/bfdc2bbfa16c2d27fa24ee0665ffb41040ef696c#diff-661a67441954052a3bb363a7c1398797R18

Other related issues: https://github.com/benmosher/eslint-plugin-import/issues/660 + 
https://github.com/benmosher/eslint-plugin-import/issues/758

According to this comment, a release should be coming soon: https://github.com/benmosher/eslint-plugin-import/issues/758#issuecomment-302306840

**Comment by benmosher** [comment]
ah yeah, I believe I just published the fix for this as v2.3.0

**Comment by kevanstannard** [comment]
Many thanks @benmosher
